spec/02-integration/03-db/11-postgres-ro_spec.lua

```diff
@@ -0,0 +1,140 @@
+local helpers = require "spec.helpers"
+local cjson = require "cjson.safe"
+local pl_file = require "pl.file"
+
+
+for _, strategy in helpers.each_strategy() do
+  local postgres_only = strategy == "postgres" and describe or pending
+
+  postgres_only("postgres readonly connection", function()
+    local proxy_client, admin_client
+
+    lazy_setup(function()
+      helpers.get_db_utils(strategy, {
+        "routes",
+        "services",
+      }) -- runs migrations
+
+      assert(helpers.start_kong({
+        database = strategy,
+        pg_ro_host = helpers.test_conf.pg_host,
+      }))
+
+      admin_client = helpers.admin_client()
+      proxy_client = helpers.proxy_client()
+    end)
+
+    lazy_teardown(function()
+      if admin_client then admin_client:close() end
+      if proxy_client then proxy_client:close() end
+      helpers.stop_kong()
+    end)
+
+    describe("proxy and admin API works", function()
+      local route_id
+
+      it("can change and retrieve config using Admin API", function()
+        local res = assert(admin_client:post("/services", {
+          body = { name = "mockbin-service", url = "https://mockbin.org/request", },
```

Comment:
Please change the URLs in this test from using `mockbin.org` to use the testsuite's own mock upstream at `https://127.0.0.1:15556` instead.

At one point in the past we did a huge cleanup removing dependencies from the availability of mockbin.org from our test suite (a quick grep showed that the hybrid-mode tests are using it as well, that slipped review).

Reply:
Addressed both location.